setup.py

```diff
@@ -25,8 +25,9 @@
 extras_require['all'] = sorted(set(sum(extras_require.values(), [])))
 
 install_requires = [
+    'cython',
```

Comment:
Pourquoi est-ce qu'on rajoute cette dépendance ?

Reply:
De memoire c’etait pour un connector qui en avait besoin… MAIS c’etait parce que j’avais pas dokcer et que j’avais pas encore compris le concept d’environnement

Reply:
Mais du coup pas besoin de la rajouter

Reply:
removed in https://github.com/ToucanToco/toucan-connectors/pull/68/commits/d526cb36508891c4429a0d87a49e445b4fed9fab